src/java/jni.rs

```diff
@@ -1,22 +1,25 @@
 //! Functions to generate JNI bindings
 extern crate inflector;
-use self::inflector::Inflector;
-use self::proc_macro2::Span;
-use super::types::{callback_name, rust_ty_to_java};
-use super::{Context, Outputs};
-use crate::common::{
-    append_output, is_array_arg, is_array_arg_barefn, is_user_data_arg, is_user_data_arg_barefn,
-    take_out_ident_from_type, take_out_pat, transform_fnarg_to_argcap,
-    transform_fnarg_to_argcap_option,
-};
-use crate::struct_field::StructField;
+
 use core::borrow::Borrow;
 use jni::signature::{self, JavaType, Primitive, TypeSignature};
 use proc_macro2;
 use quote::ToTokens;
 use quote::*;
 use unwrap::unwrap;
 
+use crate::common::{
+    append_output, is_array_arg, is_array_arg_barefn, is_user_data_arg, is_user_data_arg_barefn,
+    take_out_ident_from_type, take_out_pat, transform_fnarg_to_argcap,
+    transform_fnarg_to_argcap_option,
+};
+use crate::struct_field::StructField;
+
+use self::inflector::Inflector;
```

Comment:
Doesn't have to be `use self::<..>`, can be just this:
```suggestion
use inflector::Inflector;
```